lib/tasks_private/instance_types.rake

```diff
@@ -1,12 +1,12 @@
-desc "Create new instance_types.rb"
-task :instance_types => [:environment] do
+desc("Create new instance_types.rb")
+task(:instance_types => [:environment]) do
```

Comment:
Don't add the parens in...these are DSL methods.  If anything we should change the codeclimate.yml if that's the reason for this.  cc @bdunne

Reply:
A section for the rake gem under https://github.com/ManageIQ/guides/blob/master/.rubocop_base.yml#L75 would be great.  @jameswnl do you want to add that or should I?

Reply:
@Fryguy updated!
@bdunne yes, in this [PR](https://github.com/ManageIQ/guides/pull/249)